components/txn_types/src/lock.rs

```diff
@@ -85,6 +86,12 @@ pub struct Lock {
     // while committing is relatively expensive. So the solution is putting the ts of the rollback
     // to the lock.
     pub rollback_ts: Vec<TimeStamp>,
+
+    /// The commit TS of the latest PUT/DELETE record
```

Comment:
```suggestion
    /// The commit TS of the last PUT/DELETE record before this lock
```
It could be a bit ambiguous if the lock is not the latest record.

Reply:
I only apply this suggestion to `Write`. These two fields in the lock CF always count from the latest version. Does it make sense?

Reply:
sounds good